packages/webview_flutter/android/src/main/AndroidManifest.xml

```diff
@@ -1,2 +1,4 @@
-<manifest package="io.flutter.plugins.webviewflutter">
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+          package="io.flutter.plugins.webviewflutter">
+  <uses-sdk android:minSdkVersion="16" />
```

Comment:
This should be 20

Reply:
Just for webview, right? Are there any other plugins that should be higher than 16?

Reply:
Sorry maps as well should be 20 though it's not included in this PR. These are the ones I'm aware of.